Description: Add _Keywords to .desktop file
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Forwarded: not-needed
Last-Update: 2014-08-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/data/rapid-photo-downloader.desktop
===================================================================
--- trunk.orig/data/rapid-photo-downloader.desktop
+++ trunk/data/rapid-photo-downloader.desktop
@@ -9,3 +9,4 @@ MimeType=x-content/image-dcf;
 Terminal=false
 Type=Application
 Categories=Graphics;Photography;
+Keywords=photo;download;graphics;photography;
